roles/postgresql/tasks/backup_db.yml

```diff
@@ -0,0 +1,13 @@
+---
+# back up the database to /tmp/ on the app's VM (not the database VM)
+- name: backup postgres database
+  postgresql_db:
+    name: "{{ application_db_name }}"
+    login_host: "{{ postgres_host }}"
+    port: "{{ postgres_port  }}"
+    login_user: "{{ vault_postgres_admin_user }}"
+    login_password: "{{ vault_postgres_admin_password }}"
```

Comment:
why admin? shouldn't app user be sufficient?

Reply:
i wasn't sure if executing the dump command was a special permission. we could try it and see!

Reply:
My expectation is that if you have read access to a database, you can dump it!